Nodejs/Product/Npm/SemverVersion.cs

```diff
@@ -31,13 +31,13 @@ public struct SemverVersion {
             "^(?<major>[0-9]+)"
             + "\\.(?<minor>[0-9]+)"
             + "\\.(?<patch>[…0-9]+)" // The '…' is there to handle the 'classy' library, which has a very long version number - see slightly snarky comment about that unadulterated bag of hilarity below.

```

Comment:
Way back when... npm did not have the db stored in a json format, so we used to parse the textual output ([here's an example](https://raw.githubusercontent.com/Microsoft/nodejstools/1bb20aa7ed8b8ace307825ed7072ea54fb7ff40d/Nodejs/Tests/TestData/NpmSearchData/npmsearchfullcat_npm143.txt))

We've since [removed that code](https://github.com/Microsoft/nodejstools/commit/a1cb641ce7a9bdf7fe240d3ae069bde1f9414a2b), but looks like we missed a spot.

In general, this file needs some attention, but it's one of those areas that's been around for a while, and works well enough.